etc/contracts-test-data/contracts/loadnext/how_to_calculate_loadtest_profiles.md

```diff
@@ -0,0 +1,11 @@
+# Calculating loadtest profiles
+
+Use the SQL scripts in this folder (`query_*.sql`) to calculate the characteristics of transactions within a miniblock range.
+
+Calculate `CONTRACT_EXECUTION_PARAMS` as follows:
+
+- `light`: all zeroes.
+- `realistic`: 75th percentile.
```

Comment:
I would probably use the median since we want to measure the average mainnet transaction - but maybe I'm missing something?